classification.yaml

```diff
@@ -0,0 +1,791 @@
+  status: feature-complete
+  notes:
+    Pipeline YAML files satisfy this requirement (they don't even need to be ordered; the system can order them itself).
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0005:
+  status: feature-complete
+  notes:
+    Pipeline YAML files satisfy this requirement.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0006:
+  status: feature-complete
+  notes:
+    Grouping information is actually embedded in the PipelineTask definition, not directly the Pipeline, but since a Pipeline include PipelineTasks, this is still satisfied.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0007:
+  status: feature-complete
+  notes:
+    See notes for DMS-MWST-REQ-0006.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0008:
+  status: feature-complete
+  notes:
+    Satisfied by PipelineTask being a subclass of Task.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0003:
+  status: feature-complete
+  notes:
+    The Pipeline Python API is not heavily utilized, but it certainly does exist and was conceived of as a public interface.
+DMS-MWST-REQ-0010:
+  status: feature-complete
+  notes:
+    This is just how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0011:
+  status: feature-complete
+  notes:
+    This is just how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983f
+DMS-MWST-REQ-0012:
+  status: feature-complete
+  notes:
+    QuantumGraph generation guarantees this.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0013:
+  status: feature-complete
+  notes:
+    This is how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0009:
+  status: unclear
+  notes:
+    It looks like someone wants to delete this as redundant with -0013.  But we can do it all.
+DMS-MWST-REQ-0014:
+  status: feature-complete
+  notes:
+    Spec for this is almost impossible to understand on its own, but description makes it clear that we've satisfied this with the PipelineTask design.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0015:
+  status: unclear
+  notes:
+    I was either not aware of this one or had forgotten about it.  I am skeptical that Pipeline is the right place for this functionality, but the lack of a motivating use case makes it hard to reason about.
+DMS-MWST-REQ-0016:
+  status: feature-complete
+  notes:
+    What we have here is more of a port of the Gen2 obs-package and command-line override functionality, rather than a "generalization" of it, but I'm not sure what more the requirement wants.
+DMS-MWST-REQ-0017:
+  status: feature-complete
+  notes:
+    Satisfied by having a YAML format for Pipelines.
+DMS-MWST-REQ-0029:
+  status: unclear
+  notes:
+    I am not convinced that using butler to send alerts to the distribution system is actually the best approach, but the requirement allows for that possibility, and clearly if we *do* use the butler it can't get in the way of the performance requirements.
```

Comment:
I lean towards removing this requirement as not the right way to solve the problem, but we should make sure it isn't standing in for some other similar demand on middleware.